dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentFactoryImpl.java

```diff
@@ -2278,15 +2276,15 @@ protected void clearField(String structureInode, Field field) throws DotDataExce
         Queries queries = getQueries(field);
         List<String> inodesToFlush = new ArrayList<>();
 
-        Connection conn = DbConnectionFactory.getConnection();
+
         
-        try(PreparedStatement ps = conn.prepareStatement(queries.getSelect())) {
+        try(Connection conn = DbConnectionFactory.getConnection();PreparedStatement ps = conn.prepareStatement(queries.getSelect())) {
```

Comment:
I am not pretty sure if we should autoclose the conn here. it should be handle by the AOP

Reply:
Do you mean to use CloseDBIfOpened annotation? If so, how do we avoid the SonarLint violation?

Reply:
Yes, because you can be closing a connection wired in a previous steps, if should close conn only if needed ;)